tests/test_martingale.py

```diff
@@ -47,4 +47,8 @@ def test_stake_raised_if_bet_wins(self):
     def test_player_plays_when_stake_not_less_than_table_minimum(self):
         self.martingale.stake = 3
         playing_result = self.martingale.playing()
-        self.assertEqual(False, playing_result)
+        self.assertFalse(False)
```

Comment:
Should check `playing_result`. Better invoke `playing` in place

Reply:
Done.